lhotse/recipes/commonvoice.py

```diff
@@ -42,6 +46,8 @@
 COMMONVOICE_SPLITS = ("train", "dev", "test", "validated", "invalidated", "other")
 COMMONVOICE_DEFAULT_SPLITS = ("train", "dev", "test")
 
+set_ffmpeg_torchaudio_info_enabled(False)
```

Comment:
This shouldn't be in the global scope, because importing this file is going to have a side effect. Please move this function call to the place just before RecordingSet is created inside `prepare_commonvoice`, and then re-enable it just after the RecordingSet is constructed.

Reply:
I am thinking perhaps we can make these functions possible to be used with a context manager? Would be more "pythonic" I feel.

Reply:
+1